tensorflow/core/kernels/convex_hull_op_test.cc

```diff
@@ -0,0 +1,151 @@
+/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Nit: The spacing doesn't look right. Please add newlines around this line.
```suggestion

    http://www.apache.org/licenses/LICENSE-2.0
    
```

Reply:
change it accordingly